crates/starknet-devnet/tests/common/utils.rs

```diff
@@ -67,8 +67,8 @@ pub fn get_flattened_sierra_contract_and_casm_hash(
     (sierra_class.flatten().unwrap(), casm_hash(casm_json).unwrap())
 }
 
-pub fn get_messaging_contract_in_sierra_and_compiled_class_hash()
```

Comment:
Please don't resolve if the reviewer's comment is not addressed. There are still formatting-induced changes present in the `Files Changed` section of your PR.

In fact, it would be good to let the original comment author decide on whether a conversation is resolvable. If you just need a way to keep track of what you addressed and what you did not, you can add a reply to the conversation with a link to the commit that you believe addresses the comment.